### ISSUE TITLE
Apalache fails to find module file

### ISSUE THREAD

**Issue by craft095** [informalsystems/apalache#1477]
## Description

Let's assume following layout:

`.` <--- work directory where `java -jar apalache.jar ...` is called
`./dir/M0.tla` <-- Main module
`./dir/M1.tla` <-- `M0` extends `M1`

Then Apalache cannot find `./dir/M1.tla`. TLC works as expected.

## Input specification

```
---- MODULE M0 ----
EXTENDS M1
VARIABLE
    \* @type: Bool;
    x
Init == x = TRUE
Next == x' = ~x
Spec == Init /\ [][Next]_x
Inv == x => ~y
====
```

```
---- MODULE M1 ----
\* @type: Bool;
ap_0 == TRUE
====
```

## The command line parameters used to run the tool

`java -jar ../tools/apalache-pkg-0.20.2-full.jar check dir/M0.tla`

## Expected behavior

Apalache is able to find all tla-files

## Log files

```
# APALACHE
# version: 0.20.2
# build  : 5b59b10
#
# Usage statistics is OFF. We care about your privacy.
# If you want to help our project, consider enabling statistics with config --enable-stats=true.

Output directory: /mnt/c/work/TLA/tlc-qual/bug/_apalache-out/M0.tla/2022-03-12T21-03-30_5656217929895998894
Checker options: filename=dir/M0.tla, init=, next=, inv=          I@21:03:31.116
Tuning:                                                           I@21:03:31.122
PASS #0: SanyParser                                               I@21:03:31.124
Parsing file /mnt/c/work/TLA/tlc-qual/bug/dir/M0.tla
File does not exist: /mnt/c/work/TLA/tlc-qual/bug/M1.tla while looking in these directories: jar:file:/mnt/c/work/TLA/tlc-qual/tools/apalache-pkg-0.20.2-full.jar!/tla2sany/StandardModules/
Error by TLA+ parser: *** Abort messages: 1

Unknown location

Cannot find source file for module M1 imported in module M0.


 E@21:03:31.306
It took me 0 days  0 hours  0 min  0 sec                          I@21:03:31.307
Total time: 0.995 sec                                             I@21:03:31.308
EXITCODE: ERROR (255)
```

## System information

- Apalache version [`apalache-mc version`]: 0.20.2
- OS [e.g. Ubuntu Linux or Mac OS, and the current version]: Ubuntu 20.04
- JDK version [e.g. OpenJDK 0.8.0]:  openjdk version "14.0.2"

**Comment by shonfeder** [comment]
Thanks for the report @craft095 :)

You haven't given the full command you are running, but I assume you are not running apalache from within the `./dir` directory?

Apalache only knows how to find `.tla` modules that are either

- located in the current working directory, or,
- in directories that have been added to the `TLA_PATH`. 

Please see https://apalache.informal.systems/docs/apalache/running.html?highlight=TLA_PATH#module-lookup for details on module lookup!

**Comment by shonfeder** [comment]
As  UX improvement, we could add this info the error message, noting the use of TLA_PATH briefly and perhaps providing a link to the section in the manual.

Do you think that would be helpful?

**Comment by craft095** [comment]
Thanks @shonfeder ! I know about workaround, it just seemed weird to me that Apalache diverges from TLC in this regard. That's why I reported it. That is surely minor issue, but I find TLC behavior less surprising and more convenient, especially when invoked from script or Makefile. Is there a deeper reason why Apalache does not look for modules relative to the main tla file?

**Comment by shonfeder** [comment]
I see. Thanks for clarifying. I'll discuss this with the team at our next meeting to get the background and motivation and provide an update.

**Comment by konnov** [comment]
Related to #426. I think Markus has already mentioned a solution there.

**Comment by thpani** [comment]
@craft095 FYI, we fixed this in #1491 bound for release next Monday! Thanks again for the report!
